### PR TITLE
Move confirmAccess method to UxPerfUiAutomation

### PR DESCRIPTION
**Pull request by johnrchrdsn**
Move confirmAccess method out of reader and skype workloads into
UxPerfUiAutomation base class for code reuse.

Amend project properties for reader workload to use SDK version 18.

Update jar files appropriately.

### CODE REVIEW

File: wlauto/external/uiauto/src/com/arm/wlauto/uiauto/UxPerfUiAutomation.java [hunk 1]
```diff
@@ -267,6 +267,16 @@ public void writeResultsToFile(Map<String, Timer> results, String file) throws E
         out.close();
     }
```
Comment (johnrchrdsn) [inline wlauto/external/uiauto/src/com/arm/wlauto/uiauto/UxPerfUiAutomation.java:269]: I think this method might be used more than once in when testing on multiple devices - particularly first time runs.

File: wlauto/workloads/reader/uiauto/project.properties [hunk 2]
```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-21
```
Comment (johnrchrdsn) [inline wlauto/workloads/reader/uiauto/project.properties:14]: I assume this version was a left-over that was never cleaned up from reader.

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 3]
```diff
@@ -122,15 +122,6 @@ private void signInOnline(Bundle parameters) throws Exception {
         }
     }
 
-    private void confirmLocalFileAccess() throws Exception {
-        // First time run requires confirmation to allow access to local files
-        UiObject allowButton = new UiObject(new UiSelector().textContains("Allow")
-                                                            .className("android.widget.Button"));
-        if (allowButton.waitForExists(timeout)) {
```
Comment (johnrchrdsn) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:129]: If condtion vs while in the base class method. I don't think this should cause any problems?

File: wlauto/workloads/reader/uiauto/project.properties (hunk 2 above)
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/project.properties:14]: This should have its own commit.  Please avoid making two changes in a single commit.

**Pull request merged**